website/docs/getting-started/setup/upgrade-to-business-edition/upgrade-k8s-to-business-edition.mdx

```diff
@@ -0,0 +1,21 @@
+# 
+
+<HideElements/>
+
+Follow the steps below to upgrade Kubernetes Community edition to Business edition:
+
+1. Uninstall the community edition helm charts by running the following command:
+
+```javascript
+helm uninstall appsmith
+```
+2. Install and deploy the business edition helm charts by running the following commands:
```

Comment:
```suggestion
2. Deploy using the business edition Helm Chart by running the following commands:
```

Reply:
We can keep the step same as in the existing doc `To add and deploy the new Helm chart, run the following command:`